examples/bzlmod/.bazelrc

```diff
@@ -70,7 +73,8 @@ common:custom-linux --extra_execution_platforms=//:my_linux_platform
 common:remote-windows --config=remote
 common:remote-windows --platforms=@toolchains_buildbuddy//platforms:windows_x86_64
 common:remote-windows --extra_execution_platforms=@toolchains_buildbuddy//platforms:windows_x86_64
-
+# (Optional) Explicitly tell Bazel to use this toolchain
```

Comment:
This is a bit misleading as we don't force usage, we just register it with higher precedence. 

Since we already suggest explicit registration via the module file, I would also like to see an explanation for when you may still need the flag and that that replaces the register_toolchains call.

Reply:
Good call. I changed the comment here to direct users to the coupled `.bazelrc` file, where we have concrete examples and explanations.